stages/ui.cc

```diff
@@ -326,10 +340,17 @@ void Ui::UpdateLEDs() {
         }
         uint8_t type = configuration & 0x3;
         LedColor color = palette_[type];
+        uint16_t* seg_config = settings_->mutable_state()->segment_configuration;
```

Comment:
Bring in `state()` rather than `mutable_state()` so we don't accidentally change it. It's a const ref not a copy or anything, so no performance problem in doing that.